presto-hive/src/test/java/io/prestosql/plugin/hive/parquet/ParquetTester.java

```diff
@@ -569,7 +569,7 @@ private static FileFormat getFileFormat()
         return OPTIMIZED ? FileFormat.PRESTO_PARQUET : FileFormat.HIVE_PARQUET;
     }
 
-    private static void writeParquetColumn(
+    public static void writeParquetColumn(
```

Comment:
make it package-private?

Reply:
It's now part of the public API of this class, so package private doesn't seem appropriate.